Formula/quantlib.rb

```diff
@@ -20,6 +20,7 @@ class Quantlib < Formula
   end
 
   option :cxx11
+  option "with-intraday", "Enable intraday components to dates"
```

Comment:
Would it make sense to enable this by default? If so, would anyone want/need to remove support? If not, let's remove the option.